core/lib/generators/spree/install/install_generator.rb

```diff
@@ -58,6 +58,9 @@ def additional_tweaks
 
     def setup_assets
       @lib_name = 'spree'
+
+      empty_directory 'app/assets/images'
```

Comment:
Do you think we could add a comment here linking to this PR or an issue that explains why this is here?

Reply:
@jacobherrington I think the commit message already explains the rationale behind the change pretty well. I'm not against adding a comment, just not sure it's necessary and afraid it would unnecessarily pollute the code here. What do you think?

Reply:
I'm comfortable with that -- I think it's a good commit message. :+1: